.coveragerc

```diff
@@ -1063,6 +1063,9 @@ omit =
     homeassistant/components/telnet/switch.py
     homeassistant/components/temper/sensor.py
     homeassistant/components/tensorflow/image_processing.py
+    homeassistant/components/terncy/const.py
+    homeassistant/components/terncy/hub_monitor.py
+    homeassistant/components/terncy/light.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.